WORKFLOW.md

```diff
@@ -0,0 +1,137 @@
+X-Road Tests development
+# Workflow Policy
+
+v1.0
+
+## 1	General
+
+1.1	This document establishes workflow policy for joint development of X-Road software. Workflow Policy strives to serve as a handbook for all practical aspects of development.
+
+1.2	Workflow is understood as systematic arrangement of work, a complex of processes, practices, roles and responsibilities, communication patterns, and artefacts.
+
+1.3	Goals of the workflow policy are to
+- establish productive and secure collaborative open source working environment
+- assure production of high quality software
+- avoid duplication of effort, facilitate re-use of software
+- transparency and openness
+- broader community of developers
+- use of software development best practice
+- clear communication among Partners as well as other stakeholders in the X-Road development process
+- innovation.
+
+1.4 Adherence to this policy are made legally binding to Vendors by inclusion of appropriate stipulations in contracts between Partners and Vendors. Vendor in context of this policy is a firm or other organisation performing development work by assigment of a Partner.
+
+1.5 Partners and Developers implement this policy in good faith and in the context of sustainable, good software development practice.
+
+1.6 Projects may have their own workflow arrangements as far as these do not contradict this policy.
+
+## 2 Related documents
+
+2.1 Workflow policy is related to other X-Road tests development policy and regulations:
+- workflow policy implements Harmonized Xroad tests document [Harmonized X-Road test environment](https://github.com/ria-ee/blob/master/HARMONIZED_TEST_ENVIRONMENT.md)
```

Comment:
Xroad -> X-Road (multiple occurrence in this document)

Reply:
done